src/js/components/Layer/__tests__/__snapshots__/Layer-test.js.snap

```diff
@@ -2477,3 +2477,5 @@ exports[`Layer position top 1`] = `
 `;
 
 exports[`Layer position top 2`] = `""`;
+
+exports[`Layer should be null prior to mounting, displayed after mount 1`] = `null`;
```

Comment:
thanks for adding the test! mind just adding a nice id because right now it doesn't look like anything actionable if it breaks later!

Reply:
@oorestisime I made the test id more descriptive. Let me know if you were thinking something else.